pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -29,6 +29,11 @@ import (
 	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
+const (
+	hypervIsolationAnnotationKey   = "windows.kubernetes.io/isolation-type"
+	hypervIsolationAnnotationValue = "hyperv"
```

Comment:
Refer to const in https://github.com/moby/moby/blob/master/api/types/container/host_config.go or add a comment with link to where this string comes from?

Reply:
ack, will add a refer link